README.md

```diff
@@ -8,3 +8,61 @@ It includes Javascript and native stack traces, heap statistics,
 platform information and resource usage etc. With the report enabled,
 reports are triggered on unhandled exceptions, fatal errors, signals
 and calls to a Javascript API.
+
+Usage:
+
+    npm install nodejs/nodereport
+
+    var nodereport = require('nodereport');
+
+By default, this will trigger a NodeReport to be written to the current
```

Comment:
Perhaps an optional options arg with key `events` as an array of potential triggers and key `sigusr2` as a boolean to enable/disable the signal would be a nice approach.

Such an arg would be reasonably extensible for other triggers too.

Also allows for clear defaults to be set early in code.